dev/com.ibm.ws.jaxws.2.3.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -13,6 +13,12 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 
+webservice=JAX-WS web service configuration
+webservice.desc=Configuration for JAX-WS web services.
+
+portName=web service port name
+portName.desc=The name of the port to configure;  it is defined either in the port element of the wsdl, or the property definied in the @WebService annotation.
```

Comment:
rewrite as:
portName.desc=The name of the port to configure. This value is set in the port element in the WSDL definition or in the @WebService annotation.

Reply:
In the above comment it is not my intention to change the casing of the word WebService, the editor is changing it to all lowercase for some reason. Leave that word as you have it.

Reply:
Much better way to phrase, I'll get this changed.